portfolio-website/contactForm.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" type="text/css" href="style.css" />
+    </head>
+    <style>
+        @import url('https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css');
+    </style>
+    <body>
+        <div class="contactForm">
+            <div>
+                <span>
+                    Please fill out this contact form in order to reach me! Otherwise, click on one of the links below to find me on social media.
+                </span>
+                <form id="form">
+                        <label for="person">Name<span class="required">*</span>:</label>
+                        <input 
+                            type="text" 
+                            id="person"
+                            required
+                        > <br>
+                        <label for="email">E-mail<span class="required">*</span>:</label>
+                        <input 
+                            type="text" 
+                            id="email"
+                            required
+                        > <br>
+                        <label for="phone">Phone Number:</label>
+                        <input 
+                            type="text" 
+                            id="phone"
+                        > <br>
+                        <label for="address">Address:</label>
+                        <input 
+                            type="text" 
+                            id="address"
+                        > <br>
+                        <span class="required">*</span> means required <br>
+                        <button type="submit">Submit</button>
+                </form>
+            </div>
+            <div>
+                <p><a href="./index.html"><button id="back">Go Back</button></a></p>
+            </div>
+
+            <div class="media">
+                <p>
+                    <a href="https://www.linkedin.com/in/lsanc0043/" class="fa fa-linkedin"></a>
+                    <a href="https://twitter.com/lsanc0043" class="fa fa-twitter"></a>
+                </p>
+            </div>
+        </div>
+        <script src="cfscript.js"></script>
```

Comment:
Nice job adding another script file for a different page! And adding another page